build.gradle

```diff
@@ -777,15 +777,6 @@ task product(type: Zip) {
         exclude '.git*'
       }
     }
-    // copy AQP jar from maven repo if project not present
-    if (!hasAQPProject) {
-      copy {
-        from examplesProject.configurations.testRuntime.filter {
-          it.getName().contains('snappy-aqp')
-        }
-        into "${snappyProductDir}/jars"
-      }
-    }
```

Comment:
As per my understanding, the non-oss jars will not be available even on maven with 1.0. @kneeraj ?
So, yes, the tests requiring non-oss components currently fail (e,g, security tests, aqp examples and tests specifying memory-size), but others should be running fine. We'll need to exclude such tests from respective test suite depending upon the enterprise flag.